package.json

```diff
@@ -22,8 +22,10 @@
     "file-loader": "^1.1.11",
     "jest": "^22.4.3",
     "memory-fs": "^0.2.0",
+    "react": "^16.3.2",
```

Comment:
I don't see it being used anywhere, is it needed?

Reply:
It's needed because svgr converts the svg file to a react component.